packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -193,24 +205,136 @@ const PersonalDetailsForm = ({
                             data-testid='last_name'
                         />
                     )}
-                    {!is_appstore && !is_qualified_for_idv && !is_rendered_for_onfido && (
+                    {!is_qualified_for_idv && !is_rendered_for_onfido && !is_qualified_for_poa && (
```

Comment:
Can you please verify adding AND condition with `is_qualified_for_poa` doesn't affect this form to be rendered in other places

Reply:
Why is the additional condition needed?

Reply:
we reuse personal details component in poa form, but on design we don’t have ‘Other details’.
how this extra condition can brake other places? sure I will check it and ask QA for sanity